var/spack/repos/builtin/packages/trilinos/package.py

```diff
@@ -288,9 +288,6 @@ class Trilinos(CMakePackage, CudaPackage, ROCmPackage):
     conflicts('+stokhos', when='%xl')
     conflicts('+stokhos', when='%xl_r')
 
-    # Fortran mangling fails on Apple M1 (see spack/spack#25900)
-    conflicts('@:13.0.1 +fortran', when='target=m1')
```

Comment:
Is this now resolved for trilinos before 13.0.1?

Reply:
This error showed up in https://github.com/spack/spack/issues/25900 , and the submitter later said it was probably an issue with their environment, and I wasn't able to reproduce it. It's not known whether earlier versions of Trilinos work (or not) on M1, and if they don't it likely will be unrelated to fortran support -- so I've removed the variant.